pkg/api/scylla/validation/cluster_validation.go

```diff
@@ -216,6 +216,41 @@ func ValidateScyllaClusterSpec(spec *scyllav1.ScyllaClusterSpec, fldPath *field.
 	return allErrs
 }
 
+func ValidateRepairTaskSpec(repairTaskSpec *scyllav1.RepairTaskSpec, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	_, err := strconv.ParseFloat(repairTaskSpec.Intensity, 64)
+	if err != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("intensity"), repairTaskSpec.Intensity, "invalid intensity, it must be a float value"))
+	}
+
+	allErrs = append(allErrs, ValidateSchedulerTaskSpec(&repairTaskSpec.SchedulerTaskSpec, fldPath)...)
+
+	return allErrs
+}
+
+func ValidateBackupTaskSpec(backupTaskSpec *scyllav1.BackupTaskSpec, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	allErrs = append(allErrs, ValidateSchedulerTaskSpec(&backupTaskSpec.SchedulerTaskSpec, fldPath)...)
+
+	return allErrs
+}
+
+func ValidateSchedulerTaskSpec(schedulerTaskSpec *scyllav1.SchedulerTaskSpec, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if schedulerTaskSpec.Interval != nil {
+		if intervalDuration, err := duration.ParseDuration(*schedulerTaskSpec.Interval); err != nil {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("interval"), schedulerTaskSpec.Interval, "invalid interval format, valid units are d, h, m, s"))
```

Comment:
duplicated "invalid"

Reply:
that'll be a part of https://github.com/scylladb/scylla-operator/pull/1868, already removed there